src/libs/OptionsListUtils.ts

```diff
@@ -548,9 +552,16 @@ function getLastMessageTextForReport(report: OnyxEntry<Report>, lastActorDetails
         lastMessageTextFromReport = ReportUtils.getReimbursementDeQueuedActionMessage(lastReportAction, report, true);
     } else if (ReportActionUtils.isDeletedParentAction(lastReportAction) && ReportUtils.isChatReport(report)) {
         lastMessageTextFromReport = ReportUtils.getDeletedParentActionMessageForChatReport(lastReportAction);
-    } else if (ReportActionUtils.isPendingRemove(lastReportAction) && ReportActionUtils.isThreadParentMessage(lastReportAction, report?.reportID ?? '-1')) {
+    } else if (ReportActionUtils.isPendingRemove(lastReportAction) && ReportActionUtils.isThreadParentMessage(lastReportAction, report?.reportID ?? String(CONST.DEFAULT_NUMBER_ID))) {
         lastMessageTextFromReport = Localize.translateLocal('parentReportAction.hiddenMessage');
-    } else if (ReportUtils.isReportMessageAttachment({text: report?.lastMessageText ?? '-1', html: report?.lastMessageHtml, translationKey: report?.lastMessageTranslationKey, type: ''})) {
+    } else if (
+        ReportUtils.isReportMessageAttachment({
+            text: report?.lastMessageText ?? String(CONST.DEFAULT_NUMBER_ID),
```

Comment:
Same as above, I am not sure if I am misunderstanding the guidelines. It just feels like the empty string might be better here
```suggestion
            text: report?.lastMessageText ?? '',
```

Reply:
I think eslint check even shouldn't complain on this place, as it's not ID. 
@JKobrynski Correct me if I'm wrong.